src/main/java/org/opentripplanner/RideHailingServicesParameters.java

```diff
@@ -0,0 +1,8 @@
+package org.opentripplanner;
```

Comment:
?

Reply:
You're right, this was needlessly complex code that I removed in 16e312aca23dd8500fba42e5ebb3b171fadb75fc